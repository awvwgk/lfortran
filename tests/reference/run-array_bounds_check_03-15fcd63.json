{
    "basename": "run-array_bounds_check_03-15fcd63",
    "cmd": "lfortran --no-color {infile}",
    "infile": "tests/errors/array_bounds_check_03.f90",
    "infile_hash": "6799dc4bd57bea91b97537abefdd229857c3736e06be6a50c08655b5",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "run-array_bounds_check_03-15fcd63.stderr",
    "stderr_hash": "2f57f51e743eacdc16b23c83319a2bef7d0b15153b64408e09783dca",
    "returncode": 1
}