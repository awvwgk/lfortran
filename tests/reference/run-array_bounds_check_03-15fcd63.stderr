At 4:5 of file tests/errors/array_bounds_check_03.f90
Runtime Error: Array 'x' is not allocated.

Use '--realloc-lhs-arrays' option to reallocate LHS automatically.
